### PR TITLE
Initial setup for ERTP documentation

### PR DESCRIPTION
**Pull request by barbaraliau**
- Docs written in Markdown and built with VuePress
- Contains the initial configuration for ERTP docs
- Scripts and `deploy.sh` updated to reflect change in base folder name (typically called `/docs`)
  - `npm run docs:dev` --> `npm run docs-ertp:dev`
  - `npm run docs:build` --> `npm run docs-ertp:build`

Most of the content and formatting of the methods is changing. Much of this is set up just to show navigation and content format.

A live version is currently up at https://barbaraliau.github.io/ERTP/

### CODE REVIEW

**Comment by Chris-Hibbert** [comment]
This is a great start. Looking forward to seeing more.

File: ertp/api/descOps.md [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+galleryPixelDescOps.without([]), []);
+
+// returns [startPixel]
+galleryPixelDescOps.without([startPixel]), []);
+
+// throws error
+galleryPixelDescOps.without([]), [startPixel]);
+```
+
+# <span style="color:red">Should we move the following descriptions to the top?</span>
```
Comment (Chris-Hibbert) [inline ertp/api/descOps.md]: That would be fine.

File: ertp/guide/default-configuration.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+`strategy` - default `natStrategy`
+
+### Core Mint Keeper
+
+`mintKeeper`
+`mintKeeper.purseKeeper` - asset
+`mintKeeper.paymentKeeper` - asset
+`mintKeeper.isPurse(asset)`
+`mintKeeper.isPayment(asset)`
+
```
Comment (Chris-Hibbert) [inline ertp/guide/default-configuration.md]: These display as paragraphs rather than item lists.

**Comment by barbaraliau** [comment]
Merging as is right now.

Builds correctly, but many changes are pending, such as restructure folders to handle multiple Agoric projects, stylistic changes, additional docs, etc.

**Pull request merged**